scalding-core/src/main/scala/com/twitter/scalding/ArgHelp.scala

```diff
@@ -0,0 +1,106 @@
+package com.twitter.scalding
+
+sealed abstract class DescribedArg {
```

Comment:
Well more usually we have these all be sealed traits. Given its sealed I don't actually know if it makes a difference really.